packages/ember/tests/reexports_test.js

```diff
@@ -256,6 +255,34 @@ let allExports = [
   ['ContainerDebugAdapter', 'ember-extension-support'],
 ];
 
-if (ENV._ENABLE_PROPERTY_REQUIRED_SUPPORT) {
-  allExports.push(['required', 'ember-metal']);
+allExports.forEach(reexport => {
```

Comment:
I think you have a rebase issue here, if you notice this loop is now included above on line 10.

Reply:
Fixed